backend/src/models/notificationModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const notificationSchema = new Schema({
+    read: {
+        type: Boolean,
+        default: false
+    },
+    date: {
+        type: Date,
+        default: Date.now
+    },
+    type: {
+        type: String,
+        required: true
+    },
+    quantity: {
+        type: Number,
+        required: true
+    },
+    user: String
```

Comment:
if this is meant to be a pointer to an user, maybe it should be `user: Schema.Types.ObjectId`